en/6-use-open-standards-solutions.md

```diff
@@ -32,12 +32,54 @@ altLangPage: 6-utiliser-normes-solutions-ouvertes
 
 <section class="dpgn-section-guideline">
 
-## 6.1 Leverage open standards and embrace leading practices
+## 6.1 Leverage open standards, use open source software and embrace leading practices
 
 <div class="dpgn-section-intro-guideline">
 
 **[TODO: Add/revise introductory text]**
 
+Give equal consideration to open source software when you choose technology.
+
+Your technology project or programme could benefit from:
+
+- solving common problems with readily available open source technology
+- more time and resource for customised solutions to solve the rare or unique problems
+- lower implementation and running costs
+
+Be aware that open source software is not completely free, so take into account the total cost of migrating, including exit and transition costs.
+
+**Technology Code of Practice - Be open and use open source (UK))**
+
+Open standards can be used when designing individual elements of the solution.
+
+Using open standards means you:
+
+- save time and money by reusing things that are already available
+- increase compatibility with all stakeholders
+- potentially open up the range of companies you can purchase from as more of them are likely to use the same standard as you
+- can move between different technologies when you need to and don't get locked into contracts
+
+**(Technology Code of Practice - Make use of open standards (UK))**
+
+The city acknowledges that open source software and hardware:
+
+- entail proven benefits, namely with respect to quality, safety of information, variety of functionalities and the pooling of solutions;
+- contribute to the interoperability, reusability and interchangeability of solutions;
+- contribute to data resilience;
+- contribute to eliminating vendor lock-in;
+- constitute an industry of their own and must be assessed on merit, like every other industry.
+
+The city acknowledges that, as a public administration body, it must promote free competition in terms of software and hardware purchases.
+
+- Principle 1

```

Comment:
Principles 1, 2 and 3 seem like they could be actions that should be taken. Are there checklist items (in the Checklist section) that already cover these actions?